cranelift/codegen/src/isa/x64/inst/emit_tests.rs

```diff
@@ -2356,6 +2352,16 @@ fn test_x64_emit() {
         "F3450F5C94C241010000",
         "subss   321(%r10,%rax,8), %xmm10",
     ));
+    insns.push((
+        Inst::xmm_rm_r(SseOpcode::Mulss, RegMem::reg(xmm5), w_xmm4),
+        "F30F59E5",
+        "mulss   %xmm5, %xmm4",
+    ));
+    insns.push((
+        Inst::xmm_rm_r(SseOpcode::Divss, RegMem::reg(xmm8), w_xmm7),
+        "F3410F5EF8",
+        "divss   %xmm8, %xmm7",
+    ));
```

Comment:
Can you please add a test for sqrtss, since its emission code is filled in this PR?